### PR TITLE
Add unit tests for tpr-backed delete

### PR DESCRIPTION
**Pull request by krancour**

### CODE REVIEW

File: pkg/storage/tpr/storage_interface_test.go [hunk 1]
```diff
@@ -361,6 +360,37 @@ func TestDeleteNonExistent(t *testing.T) {
 	}
 }
 
+func TestDelete(t *testing.T) {
```
Comment (pmorie) [inline pkg/storage/tpr/storage_interface_test.go]: Looks good - how about another test (can be in a different PR, I know you are very busy this week) that does a delete on an object that doesn't exist?

Comment (krancour) [inline pkg/storage/tpr/storage_interface_test.go]: That test already exists.

Comment (pmorie) [inline pkg/storage/tpr/storage_interface_test.go]: Cool

**Comment by arschles** [comment]
I am merging this on good faith, as it adds additional unit tests and nothing else
